app/Http/Controllers/UserListController.php

```diff
@@ -29,7 +29,7 @@ public function store(Request $request){
         return \Redirect::back();
     }
 
-    public function delete_game($listid, $itemid){
+    public function delete_game($listid, $itemid) {
```

Comment:
__[code_sniffer]__
* Method name "UserListController::delete_game" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->